projects/ngneat/reactive-forms/src/lib/control-actions.ts

```diff
@@ -122,7 +130,7 @@ export function hasErrorAndDirty<T>(control: AbstractControl<T>, error: string,
 
 export function markAllDirty<T>(control: FormArray<T> | FormGroup<T>): void {
   control.markAsDirty({ onlySelf: true });
-  (control as any)._forEachChild(control => control.markAllAsDirty());
+  (control as any)._forEachChild(control => control.markAllAsDirty?.());
```

Comment:
Why did you add this?

Reply:
I got an exception when using the form array factories. I'll recheck why exactly that happened.